serverless.yml

```diff
@@ -0,0 +1,108 @@
+service: bucket-antivirus-function
+
+provider:
+  name: aws
+  runtime: python3.7
+  region: ap-southeast-2
+  timeout: 300
+
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action:
+        - "s3:GetObject"
+        - "s3:GetObjectTagging"
+        - "s3:PutObject"
+        - "s3:PutObjectTagging"
+        - "s3:PutObjectVersionTagging"
+      Resource: { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : "S3BucketClamavVirusDatabase" }, "/*" ] ]  }
+
+    - Effect: "Allow"
+      Action:
+        - "s3:ListBucket"
+      Resource:
+        - { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : "S3BucketClamavVirusDatabase" }, "/*" ] ]  }
+        - { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : "S3BucketClamavVirusDatabase" } ] ]  }
+
+    - Effect: "Allow"
+      Action:
+        - "s3:GetObject"
```

Comment:
Is this allowing getting objects from any bucket? If so, should we restrict this?

Reply:
Added a list of s3 buckets in the stack parameters